torch/csrc/jit/codegen/cuda/ir_nodes.cpp

```diff
@@ -1223,7 +1223,12 @@ class ConcretizeDomain : private BackwardVisitor {
 };
 
 void ConcretizeDomain::concretizePwOp(Expr* e) {
-  TensorView* tv = *ir_utils::filterByType<TensorView>(e->outputs()).begin();
+  if (e->output(0)->getValType() != ValType::TensorView) {
```

Comment:
I would prefer using ir_utils::isTVOp, which includes all related checks.

Reply:
Thanks. I thought about that, but `isTVOp` is in lower_utils.h, which I didn't want to include from ir_nodes.cpp. We should move utility functions not tied to lowering, like `isTVOp`, to ir_utils.h.